modules/syslogformat/syslog-format.c

```diff
@@ -1030,6 +1030,7 @@ log_msg_parse_legacy(const MsgFormatOptions *parse_options,
       goto error;
     }
 
+  msg_debug("Setting macros on BSD syslog message");
   log_msg_parse_seq(self, &src, &left);
```

Comment:
What's the added value of this message? I guess there's an "initial parse starts" message, isn't there?

Reply:
Also, please don't use the "macros" term.

Reply:
Thinking it again there is no additional value in this message, I will delete it.
It was just good to know if there was a legacy message parsing or syslog message parsing.